### ISSUE TITLE
Code block not displayed if EOL char is CRLF

### ISSUE THREAD

**Issue by enr** [charmbracelet/glow#143]
Hi, thank you for this tool.
I noticed a problem in the display of the code blocks when the end of line char is set to the Windows' one (CRLF).

(Probably, this issue should be redirected to the Markdown library Glow is using)

To reproduce just try this text:

```
    # test

    test

    ```
    test test 
    ```
```
Cheers, Enrico

**Comment by muesli** [comment]
Indeed, there's an issue with the syntax highlighter not correctly interpreting CRLFs. I've already opened a ticket upstream and work's underway to fix this (in the right place): https://github.com/alecthomas/chroma/pull/336

Hoping to see this merged soon!
